Library/Input_Configuration/FineTuned/FineTuned_FWD/OPTIONS_FineTuned_LMM_FWD.m

```diff
@@ -0,0 +1,60 @@
+%% OPTIONS_FineTuned_ROK_FWD
+%
+% <html>
+%   <div>
+%       <img style="float: right" src="../../../../../MATLODE_LOGO.png" height="150px"></img>
+%   </div>
+% </html>
+%
+% <html>
+% Up: <a href="../../../../html/Library.html">Library</a>
+% </html>
+%
+%% Rosenbrock Krylov (ROK) Forward (FWD) Fine Tuning
```

Comment:
The whole comment is copied from the ROK finetuned file, but this is the LMM file.